### PR TITLE
Add tracetronic to showcase

### PR DESCRIPTION
**Pull request by vinzBad**
Had a great time lektorifying the work of our UX designers. I think the result speaks for itself.

### CODE REVIEW

**Comment by nixjdm** [comment]
Nice looking site! Thanks! Would you mind redoing the second image though without the browser chrome?

**Comment by vinzBad** [comment]
Whoops, I‘ll fix it this evening

**Comment by nixjdm** [comment]
Thanks!

**Comment by vinzBad** [comment]
I've updated my branch.

File: content/showcase/tracetronic/contents.lr [hunk 1]
```diff
@@ -0,0 +1,8 @@
+name: TraceTronic
+---
+description:
+We wanted to redesign our website and chose Lektor because it combines the ease of use for regular content editors with the strenghts of static site generators. Also we like Python ;)
```
Comment (runfalk) [inline content/showcase/tracetronic/contents.lr]: Maybe you want `strengths` not `strenghts`? :smile:

Comment (vinzBad) [reply content/showcase/tracetronic/contents.lr]: thanks :) i've fixed it

**Comment by vinzBad** [comment]
Updated my branch again to fix the typo noticed by @runfalk

**Pull request merged**